fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -291,6 +301,9 @@ protected void parseArgs(String[] args) {
             case "--dry-run":
                 this.dryRun = Boolean.TRUE;
                 break;
+            case "--batch-db-only":
+                this.batchOnly = Boolean.TRUE;
+                break;
```

Comment:
whats the intent of this one?  why not just do it lock-step with the update-schema like we're doing for the OAuth schema?

Reply:
if it really needs to be separate, maybe we should split it into three actions?
* `--update-schema`
* `--update-batch-schema`
* `--update-oauth-schema`

Like have those set the `createFhirSchema`, `createOauthSchema`, and `createJavaBatchSchema` flags respectively, then base the update on those flags alone...

Reply:
when i did the oauth schema, i was going to do something like that until i decided that the separate actions weren't really needed and so i just did the oauth ones as part of the normal create and update schema actions

Reply:
After discuss with @lmsurpre , we agree that both oauth and batch has the same issue. and we'd better allow to use different databases for batch and oauth which is/are shared by the whole service. so the wanted changes are: admin always created with each database; and admin keep info for schemas in the same database; not matter they are fhirbath, oauth or fhirdata; and fhirbatch, oauth or fhirdata schema can be updated seperately.

Reply:
I'm not going to drop it - I'm just going to change to match Lee's request. 
I'd rather do this all at once, it's an easyish change

Reply:
sound good! thanks!

Reply:
Paul added separate actions for
```
    --update-schema-fhir SCHEMANAME
    --update-schema-batch SCHEMANAME
    --update-schema-oauth SCHEMANAME
```
while preserving the existing behavior of `--update-schema`.

I think the updated design manages to walk the fine line of backwards-compatibility while still adding the requested functionality.